api/run.py

```diff
@@ -502,7 +502,8 @@ def getStudentProfileForm():
         'blank_form_data': blank_form_data,
         'basic_info': studentsDOM.getBasicInfo(studentID),
         'parent_profile': cleaned_parent_profile,
-        'form_info': form_info
+        'form_info': form_info,
+        'status': form_info['completed'],
```

Comment:
If completed is inside form_info, do we need to pass it in as a separate field? Or can we have the front end parse it as data.form_info.completed

Reply:
I'm okay with either just want to understand thought process